client/app/components/packages/project-geography.js

```diff
@@ -19,35 +20,49 @@ export default class ProjectGeographyComponent extends Component {
   @service
   store;
 
+  @tracked bblAlreadyCreated = false;
```

Comment:
Can you think of another approach that doesn't require adding more state to the component?

For example, your code in the selectSearchResult method could trigger a call to `alert` with a simple message.